centreon/tests/e2e/features/Resources-Access-Management/02-rules-management/index.ts

```diff
@@ -0,0 +1,386 @@
+  cy.visit(`centreon/monitoring/resources`);
+  cy.waitUntil(
+    () => {
+      return cy.get('div[class$="-root-emptyDataCell"]').then(($element) => {
+        return cy.wrap($element.length === 1);
+      });
+    },
+    { interval: 5000, timeout: 100000 }
+  );
+  cy.get('div[class$="-root-emptyDataCell"]').contains('No result found');
+  cy.logoutViaAPI();
+});
+
+Given('an Administrator is logged in on the platform', () => {
+  cy.loginByTypeOfUser({ jsonName: 'admin' });
+});
+
+When('a new host is created', () => {
+  cy.addHost({
+    hostGroup: 'Linux-Servers',
+    name: services.serviceOk.host,
+    template: 'generic-host'
+  })
+    .addService({
+      activeCheckEnabled: false,
+      host: services.serviceOk.host,
+      maxCheckAttempts: 1,
+      name: services.serviceOk.name,
+      template: services.serviceOk.template
+    })
+    .addService({
+      activeCheckEnabled: false,
+      host: services.serviceOk.host,
+      maxCheckAttempts: 1,
+      name: 'service2',
+      template: services.serviceWarning.template
+    })
+    .addService({
+      activeCheckEnabled: false,
+      host: services.serviceOk.host,
+      maxCheckAttempts: 1,
+      name: services.serviceCritical.name,
+      template: services.serviceCritical.template
+    })
+    .applyPollerConfiguration();
+
+  cy.addHost({
+    hostGroup: 'Linux-Servers',
+    name: services.serviceCritical.host,
+    template: 'generic-host'
+  })
+    .addService({
+      activeCheckEnabled: false,
+      host: services.serviceCritical.host,
+      maxCheckAttempts: 1,
+      name: services.serviceOk.name,
+      template: services.serviceOk.template
+    })
+    .addService({
+      activeCheckEnabled: false,
+      host: services.serviceCritical.host,
+      maxCheckAttempts: 1,
+      name: 'service2',
+      template: services.serviceWarning.template
+    })
+    .addService({
+      activeCheckEnabled: false,
+      host: services.serviceCritical.host,
+      maxCheckAttempts: 1,
+      name: services.serviceCritical.name,
+      template: services.serviceCritical.template
+    })
+    .applyPollerConfiguration();
+  checkHostsAreMonitored([
+    { name: services.serviceOk.host },
+    { name: services.serviceCritical.host }
+  ]);
+  checkServicesAreMonitored([
+    { name: services.serviceCritical.name },
+    { name: services.serviceOk.name }
+  ]);
+  cy.submitResults(resultsToSubmit);
+  checkServicesAreMonitored([
+    { name: services.serviceCritical.name, status: 'critical' },
+    { name: services.serviceOk.name, status: 'ok' }
+  ]);
+  cy.visit(`centreon/monitoring/resources`);
+  cy.get('#Hosts-button').click();
+  cy.get('#Hosts-menu').within(() => {
+    cy.contains('All').click();
+  });
+  cy.wait('@getAllResourcesStatus');
+  cy.contains('Centreon-Database').should('be.visible');
+  cy.contains('Centreon-New').should('be.visible');
+});
+
+Then(
+  'the Administrator is redirected to the "Resource Access Management" page',
```

Comment:
the administrator seems not redirected because you visit manually the page
```suggestion
  'the Administrator visits the "Resource Access Management" page',
```